app/views/warehouse_reports/health/claims/running.js.coffee

```diff
@@ -1 +0,0 @@
-$('.report-listing').html "<%=j render 'table' %>"
```

Comment:
Was this deletion intentional?  I thought this was used while the report was running.

Reply:
It was intentional, it's dead code